src/App.js

```diff
@@ -1,6 +1,11 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
```

Comment:
- [ ] It would be nice to not disable Linter checks as they are there to help us write clean and professional code. Kindly fix the error being flagged so your code is professional and free from Linter errors. 👍🏽